bindings/nodejs/lib/types/feature.ts

```diff
@@ -89,5 +89,20 @@ class TagFeature extends Feature {
         return this.tag;
     }
 }
-
-export { Feature, SenderFeature, IssuerFeature, MetadataFeature, TagFeature };
+const FeatureDiscriminator = {
+    property: 'type',
+    subTypes: [
+        { value: SenderFeature, name: FeatureType.Sender as any },
+        { value: IssuerFeature, name: FeatureType.Issuer as any },
+        { value: MetadataFeature, name: FeatureType.Metadata as any },
+        { value: TagFeature, name: FeatureType.Tag as any },
```

Comment:
Can you explain what's happening here ?

Reply:
needed for properly deserialising sub-classes with json. otherwise theyll all be instance of the base class `Feature`. Its used in [class-transformer](https://github.com/typestack/class-transformer)